### DISCUSSION TITLE
Is it possible to scale content to fit page size?

### DISCUSSION THREAD

**Discussion by martinjonsson01** [QuestPDF/QuestPDF#637] [Q&A] [answered]
Is there a way to simply scale everything up and down to the given page size? My scenario is that I've got a specific hardcoded design for A4, and for sizes A6-A1 I'd simply like the exact same A4-proportions to be scaled up or down.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

The [Scale](https://www.questpdf.com/api-reference/scale.html) element addresses your use case — it proportionally scales all inner content (text, images, padding, etc.), allowing you to maintain a hardcoded A4 layout and reproduce it at any other page size.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
